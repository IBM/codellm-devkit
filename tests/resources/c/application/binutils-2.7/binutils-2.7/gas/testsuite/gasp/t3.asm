
	test base convertions

	B'1001000
	Q'210
	D'136
	H'88

	FOOB'1001000BAR
	FOOQ'210BAR
	FOOD'136BAR
	FOOH'88BAR
