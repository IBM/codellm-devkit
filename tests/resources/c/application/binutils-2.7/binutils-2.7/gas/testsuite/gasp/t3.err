END missing from end of file.
