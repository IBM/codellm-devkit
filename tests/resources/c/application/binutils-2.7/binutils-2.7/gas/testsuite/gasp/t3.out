!

!	test base convertions
	test base convertions
!

!	B'1001000
	72
!	Q'210
	136
!	D'136
	136
!	H'88
	136
!

!	FOOB'1001000BAR
	FOOB'1001000BAR
!	FOOQ'210BAR
	FOOQ'210BAR
!	FOOD'136BAR
	FOOD'136BAR
!	FOOH'88BAR
	FOOH'88BAR
!